config/docker/docker-compose.yml

```diff
@@ -90,4 +90,8 @@ services:
     volumes:
       - "./web-ui.json:/usr/share/nginx/html/opfab/web-ui.json"
       - "./ui-menu.json:/usr/share/nginx/html/opfab/ui-menu.json"
-      - "./nginx.conf:/etc/nginx/conf.d/default.conf"
+      - "./ngnix.conf:/etc/nginx/conf.d/default.conf"
+  ext-app:
```

Comment:
Add following comment in file : 
#external application example, must not be activated in production mode